### PR TITLE
Update to kubernetes-1.15.4

### PR DESCRIPTION
**Pull request by sslavic**
Towards giantswarm/giantswarm#6302

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -38,15 +38,15 @@ required = [
   name = "github.com/docker/distribution"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```
Comment (sslavic) [inline Gopkg.toml]: This is to be reverted once https://github.com/giantswarm/apiextensions/pull/287 gets merged.

File: Gopkg.toml [hunk 2]
```diff
@@ -38,15 +38,15 @@ required = [
...
   name = "github.com/giantswarm/apiextensions"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```
Comment (sslavic) [inline Gopkg.toml]: This is to be reverted once https://github.com/giantswarm/certs/pull/48 gets merged.

File: Gopkg.toml [hunk 3]
```diff
@@ -38,15 +38,15 @@ required = [
...
   name = "github.com/giantswarm/certs"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```
Comment (sslavic) [inline Gopkg.toml]: This is to be revered once https://github.com/giantswarm/helmclient/pull/119 gets merged.

File: Gopkg.toml [hunk 4]
```diff
@@ -58,16 +58,36 @@ required = [
   name = "github.com/giantswarm/micrologger"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```
Comment (sslavic) [inline Gopkg.toml]: This is to be reverted once https://github.com/giantswarm/operatorkit/pull/319 gets merged.

**Review by tomahawk28 — APPROVED** [review]
LGTM

File: Gopkg.toml [hunk 5]
```diff
@@ -58,16 +58,36 @@ required = [
...
   name = "k8s.io/apiserver"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
+
+[[override]]
```
Comment (sslavic) [inline Gopkg.toml:72]: These overrides enforce transitive dependency versions so `dep ensure -update` can complete resolving dependencies and of appropriate version.

**Pull request merged**